tests/test_scheduler.py

```diff
@@ -35,6 +35,21 @@ def test_get_jobs_to_enqueue(self):
         self.assertEqual(registry.get_jobs_to_enqueue(timestamp + 20),
                          ['foo', 'bar'])
 
+    def test_get_jobs_to_schedule_with_chunk_size(self):
+        """Max amount of jobs returns by get_jobs_to_schedule() equal to chunk_size"""
+        queue = Queue(connection=self.testconn)
+        registry = ScheduledJobRegistry(queue=queue)
+        timestamp = current_timestamp()
+        chunk_size = 1000
```

Comment:
To make the test lighter to run, let's just change `chunk_size` to 5 or 10.

Reply:
Thanks, done